object/search.go

```diff
@@ -231,22 +241,27 @@ func (f *SearchFilters) addFlagFilter(keyTyp filterKeyType) {
 	f.addFilter(MatchUnknown, keyTyp, "", staticStringer(""))
 }
 
+// AddObjectVersionFilter adds a filter by version.
 func (f *SearchFilters) AddObjectVersionFilter(op SearchMatchType, v version.Version) {
 	f.addReservedFilter(op, fKeyVersion, staticStringer(version.EncodeToString(v)))
 }
 
+// AddObjectContainerIDFilter adds a filter by container id.
 func (f *SearchFilters) AddObjectContainerIDFilter(m SearchMatchType, id cid.ID) {
 	f.addReservedFilter(m, fKeyContainerID, id)
 }
 
+// AddObjectOwnerIDFilter adds a filter by user id.
 func (f *SearchFilters) AddObjectOwnerIDFilter(m SearchMatchType, id user.ID) {
 	f.addReservedFilter(m, fKeyOwnerID, id)
 }
 
+// AddNotificationEpochFilter adds a filter by epoch.
```

Comment:
i would highlight that epoch is not about expiration (or session, or any other epoch meaning that is wrong here) but about notification production

Reply:
added